Rakefile

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
+# make continuous integration using rubygem-packaging_rake_tasks
+# Copyright © 2015 SUSE
+# MIT license
+
+require "packaging/tasks"
+require "packaging/configuration"
+# skip 'tarball' task, it's redefined here
+Packaging::Tasks.load_tasks(:exclude => ["tarball.rake"])
+
+Packaging.configuration do |conf|
+  conf.obs_project    = "YaST:Head"
+  conf.obs_sr_project = "openSUSE:Factory"
+  conf.package_dir    = ".obsdir"
+  conf.skip_license_check << /.*/
+end
```

Comment:
This config ([API docs here](http://www.rubydoc.info/github/openSUSE/packaging_tasks/Packaging/Configuration)) is used by `rake osc:commit` and `rake osc:sr`. Actually these tasks are not used on ci.suse.de now (only `rake osc:build` is), as I did not want automatic submission before we have agreed on these PRs.